compiler/src/parsing/parser.messages

```diff
@@ -3285,14 +3297,76 @@ program: LBRACE UIDENT COMMA UIDENT COMMA EOL WHILE
 ## may provide an INCOMPLETE view of the future (what was expected next).
 ## In state 3, spurious reduction of production nonempty_list(eol) -> EOL
 ## In state 5, spurious reduction of production eols -> nonempty_list(eol)
-## In state 96, spurious reduction of production comma -> COMMA eols
+## In state 94, spurious reduction of production comma -> COMMA eols
+##
+program: LBRACE ELLIPSIS BIGINT COMMA ELLIPSIS BIGINT ARROW
```

Comment:
Technically I think we're looking for a comma followed by more fields or a }.

Reply:
Hmm, I didn't actually write these programs. After making changes to `parser.mly` I ran `npm run parser:check-errors` and these programs were autogenerated by Menhir.

Reply:
Upon re-reading this I think I may have misunderstood what you meant: are you saying that the error message should indicate that more fields or } should be given or are you saying that this is what should be in the program on the line highlighted?

Reply:
Yeah, just that the error message here isn't _quite_ right for the input program.

Reply:
If you didn't change anything in this file then it probably was already a bug and we can just make an issue for it :)

(by the way, are you sure you didn't run `npm run parser:update-errors`? `check-errors` shouldn't have made any changes to this file).

Reply:
Oh yeah I did run `parser:update-errors`, I forgot to mention that. But yes you're right about this message; originally I wrote the parser to treat the expression after ellipsis differently from fields but now that this distinction is made in `ast_helper` this should have changed.